### PR TITLE
Fixed transient issue on callback handler

### PR DESCRIPTION
**Pull request by ChetanGN**

### CODE REVIEW

File: woo-razorpay.php [hunk 1]
```diff
@@ -939,10 +946,20 @@ public function getRazorpayApiInstance()
...
         {
             global $woocommerce;
+            global $wpdb;
 
-            $orderId = get_transient(self::SESSION_KEY);
+            $order = false;
 
-            $order = wc_get_order($orderId);
+            $post_type = 'shop_order';
+            $post_password = $_GET['order_key'];
```
Comment (ramth05) [inline woo-razorpay.php:954]: Please check if sanitisation required here.

File: woo-razorpay.php [hunk 2]
```diff
@@ -939,10 +946,20 @@ public function getRazorpayApiInstance()
         function check_razorpay_response()
         {
             global $woocommerce;
```
Comment (ramth05) [inline woo-razorpay.php:948]: are we using this one.

**Review by jampulanaveen — APPROVED** [review]
LGTM

**Review by sonyy-m — APPROVED** [review]
LGTM

File: woo-razorpay.php (hunk 1 above)
Comment (ramth05) [reply woo-razorpay.php:954]: @ChetanGN , please check if this check required.

**Pull request merged**